src/modules/global/components/MenuDropdownMobile.tsx

```diff
@@ -15,11 +16,14 @@ export default function MenuDropdownMobile({
 }: Props) {
   return (
     <ul
-      className={`${
+      className={clsx(
+        "md:hidden absolute h-[calc(100vh-80px)] md:h-[100px] md:right-[40px] top-full w-full md:w-[20%] 2xl:w-[15%] bg-basic-white z-20",
         isMenuOpen
           ? "animate-navbarMobileDropdownAnimation"
-          : "animate-navbarMobileDropdownAnimationReverse"
-      } md:hidden absolute h-[calc(100vh-80px)] md:h-[100px] md:right-[40px] top-full w-full md:w-[20%] 2xl:w-[15%] bg-basic-white z-20`}
+          : !isMenuOpen
```

Comment:
No need for the 2nd condition.

Reply:
Added isMenuAnimated state from Navbar, need it to prevent navbarMobileDropdownAnimationReverse from running while resizing.